addons/calendar/models/calendar.py

```diff
@@ -0,0 +1,1047 @@
+    recurrency = fields.Boolean('Recurrent', help="Recurrent Meeting")
+    recurrent_id = fields.Integer('Recurrent ID')
+    recurrent_id_date = fields.Datetime('Recurrent ID date')
+    end_type = fields.Selection([('count', 'Number of repetitions'), ('end_date', 'End date')], 'Recurrence Termination', default='count')
+    interval = fields.Integer('Repeat Every', default=1, help="Repeat every (Days/Week/Month/Year)")
+    count = fields.Integer('Repeat', default=1, help="Repeat x times")
+    mo = fields.Boolean('Mon')
+    tu = fields.Boolean('Tue')
+    we = fields.Boolean('Wed')
+    th = fields.Boolean('Thu')
+    fr = fields.Boolean('Fri')
+    sa = fields.Boolean('Sat')
+    su = fields.Boolean('Sun')
+    month_by = fields.Selection([('date', 'Date of month'), ('day', 'Day of month')], string='Option', default='date', oldname='select1')
+    day = fields.Integer('Date of month')
+    week_list = fields.Selection([('MO', 'Monday'), ('TU', 'Tuesday'), ('WE', 'Wednesday'), ('TH', 'Thursday'), ('FR', 'Friday'), ('SA', 'Saturday'), ('SU', 'Sunday')], 'Weekday')
+    byday = fields.Selection([('1', 'First'), ('2', 'Second'), ('3', 'Third'), ('4', 'Fourth'), ('5', 'Fifth'), ('-1', 'Last')], 'By day')
+    final_date = fields.Date('Repeat Until')  # The last event of a recurrence
+
+    user_id = fields.Many2one('res.users', 'Responsible', default=lambda self: self.env.uid, states={'done': [('readonly', True)]})
+    color_partner_id = fields.Many2one(related='user_id.partner_id', string="Color index of creator")  # Color of creator
+    active = fields.Boolean(default=True, help="If the active field is set to false, it will allow you to hide the event alarm information without removing it.")
+    categ_ids = fields.Many2many('calendar.event.type', 'meeting_category_rel', 'event_id', 'type_id', 'Tags')
+    attendee_ids = fields.One2many('calendar.attendee', 'event_id', 'Attendees', ondelete='cascade')
+    partner_ids = fields.Many2many('res.partner', 'calendar_event_res_partner_rel', string='Attendees', default=_get_default_partners, states={'done': [('readonly', True)]})
+    alarm_ids = fields.Many2many('calendar.alarm', 'calendar_alarm_calendar_event_rel', string='Reminders', ondelete="restrict", copy=False)
+
+    def _compute_attendee_status(self):
+        for meeting in self:
+            attendee = meeting._find_my_attendee()
+            meeting.attendee_status = attendee.state if attendee else 'needsAction'
+
+    def _compute_display_time(self):
+        for meeting in self:
+            meeting.display_time = self._get_display_time(meeting.start, meeting.stop, meeting.duration, meeting.allday)
+
+    @api.depends('start_date', 'allday', 'start_datetime')
+    def _compute_display_start(self):
```

Comment:
You can keep single function for _compute_display_start and _compute_start as both having same stuff